packages/fast-components/src/text-area/text-area.template.ts

```diff
@@ -0,0 +1,38 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { TextArea } from "./text-area";
+
+export const TextAreaTemplate = html<TextArea>`
+    ${when(
+        x => x.childNodes.length,
+        html`
+        <label
+            part="label"
+            class="label"
+            for="control"
+        >
+            <slot name="label"></slot>
+        </label>
+    `
+    )}
+    <textarea
+        part="control"
+        class="control"
+        id="${x => (x.childNodes.length ? "control" : void 0)}"
```

Comment:
Any reason not to just give it an id either way?

Reply:
No, just thinking if it's not used it's not needed, but we can add either way.